server/planning/planning.py

```diff
@@ -69,13 +80,158 @@ class PlanningService(superdesk.Service):
     },
 
     # Planning Details
-    # NewsML-G2 Event properties See IPTC-G2-Implementation_Guide 16.5.1
+    # NewsML-G2 Event properties See IPTC-G2-Implementation_Guide 16
     # probably can skip this subsection, although its documented in iptc impl guide this way
-    'planning_details': {
+    'planning_item': {
         'type': 'dict',
         'schema': {
-            # TODO: fill in from NewsML-G2 version 2.21 spec
-        }
+            # See IPTC-G2-Implementation_Guide 16.1
+            'item_meta': {
```

Comment:
the way I understand how this works is that these are the values that will be defaulted for any items/content added as coverage to this planning item, right?  so here you think we should copy the newsml related fields we have currently in superdesk-core/superdesk/metadata/item.py, yeah?  where it makes sense, of course.